source/api/annex/notes/editors.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "IIIF Editorial Process"
+layout: spec
+tags: [annex, presentation-api, image-api]
+cssversion: 2
+---
+
+## Status of this Document
+{:.no_toc}
+
+This document is not subject to [IIIF's versioning semantics][iiif-semver]. Changes will be tracked within the document and its [internal change log][change-log]
+
+**Authors:**
+
+  * Michael Appleby, _Yale University_
+  * Tom Crane, _Digirati_
+  * Robert Sanderson, _Stanford University_
+  * Jon Stroop, _Princeton University_
+  * Simeon Warner, _Cornell University_
+  {: .names}
+
+{% include copyright.md %}
+
+----
+
+## Contents
+{:.no_toc}
+* Table of Discontent (will be replaced by macro)
+{:toc}
+
+## 1. Overview
+
+The IIIF editors are responsible for facilitating discussions across the IIIF community, digesting those discussions into use cases, and reflecting those use cases into new and revised specifications for the community to implement. This process is intended to be as transparent as possible. This document exists to further that intention by defining the IIIF editorial process and the responsibilities and expectations of the individual editors.
+
+## 2. Community Process
+
+### 2.1 Proposals and Feedback
+
+Community members should propose and discuss features and changes via [IIIF-Discuss][iiif-discuss]. However, there are a few other ways in which the IIIF Community can reach out to the editors. In particular, the IIIF holds a bi-weekly community conference call, the details and agenda for which are announced on IIIF-Discuss. Additionally, the [iiif.io issues on Github][iiif-github-issues] are open, and are especially useful for implementers who want to track the editorial discussion of a specific issue. Finally, conversations with individual editors are also possible, but are the least practical way to propose changes because the conversation will then have to be repeated to allow community input through one or more of the channels listed above.
+
+### 2.2 Use Cases and Support
+
+New specifications and changes to existing specification must have one or more [documented use cases][use-cases], supported by at least two institutions that either have IIIF-based technology in production or are clearly blocked from doing so, as demonstrated by the use case. In the case of breaking changes, where applicable, the documented use case must clearly demonstrate why the previous approach was flawed or insufficient.
+
+### 2.3 Evaluation and Testing
+
+In order to be considered ready for review, new features must have two open-source server-side implementations, at least one of which should be in production. New features must also have at least one open-source client-side implementation, which may be a proof-of-concept.
```

Comment:
It occurs to me now that this is really about new versions of specs. We can put something into version n+1 draft, but we can't ratify a new version of a spec until the changes have reference implantations.